sessions.py

```diff
@@ -0,0 +1,49 @@
+from collections import Counter
+
+def hash_session(session_response):
+    # Here we'll use all "significant" fields of sessions
+    # and form a tuple out of them. A tuple can then be hashed
+    # to give a unique hash to this session.
+    # Goal is for hash to change if any field that we care about
+    # in a session changes. 
+    data = \
+        tuple((session_response.get(s, None) for s in (
+            "center_id", "from", "to", "fee_type", "session_id",
+            "date", "available_capacity", "min_age_limit", "vaccine"
```

Comment:
Umm, so for `availability`, do we really care if it changed from `N` to `N-x` as long as `availability>0`?

Reply:
ah you're right - this shouldn't matter. Will change that.